base/math.jl

```diff
@@ -553,7 +567,9 @@ julia> asind.((0, 1/2, 1))
 asin(x::Number)
 
 """
-    acos(x)
+    acos(x::T) where T <: Number -> float(T)
+
+Returns a `T(NaN)` if `isnan(x)`.
```

Comment:
Kind of weird to have this before the description of what the function actually does.